lib/keypair.ex

```diff
@@ -7,6 +7,8 @@ defmodule Elixium.KeyPair do
   @sigtype :ecdsa
   @curve :secp256k1
   @hashtype :sha256
+  @store Elixium.Utilities.choose_store("keys","test_keys")
```

Comment:
Is there any way we can make this less intrusive? I can imagine it getting cumbersome to have to do this for every new store we have

Reply:
Perhaps be changing the :data_path in config based on whether the env is test or not? That way we can leave the store names the same and just have the tests create a dir like ~/.elixium_test/ or something